documents/policies/external/Competition Requirements.md

```diff
@@ -97,28 +99,52 @@ The WCA Delegate must submit the following information when requesting approval
       2. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
-   3. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
+   3. The deadline for updating registered events for competitors with completed registrations must be outlined.
+      1. This deadline must not be earlier than the time registration is set to close. 
+         1. The deadline may be set to be just before an event is scheduled to take place. 
+      2. Competitors may choose not to compete in an event they are registered for.
+   4. The organization team must outline the conditions of the waiting list.
+      1. If a waiting list is in place:
+         1. Registrants will be placed on the waiting list if they have completed all requirements for a valid registration, but the competitor limit has been filled. Registrants from the waiting list will be accepted in the order that their registration was completed once a spot becomes available.
+         2. It must be stated what the deadline is to be accepted for a registrant on the waiting list.
+            1. This deadline should be the same as the deadline for refunds for competitors canceling their registration, if (partial) refunds are to be given.
+         3. The organization team may specify a limit to the waiting list. Once the limit of the waiting list has been reached, the Delegate should inform the WCAT to close the registration period. 
+            1. In case a limit was not set in advance, the Delegate may contact the WCAT to request to have the registration period closed if the waiting list has become unreasonably long.
+   5. Different locations for multi-location competitions may have different registration requirements (e.g. competitor limit).
 7. Events
    1. Each event must have a planned number of rounds.
       1. Each round of each event must have a planned format and time limit.
       2. A round of an event must have planned advancement conditions if there is a subsequent round of the event.
       3. A round may have a cutoff. A round with a cutoff is a combined round, with the formats as per [WCA Regulation 9b)](wca{regulations/#9b}).
    2. Only results from events that appear on the competition website in the Events tab and Schedule will be recognized.
       1. Other events may only be held unofficially.
-   3. If the main event of the competition is not the 3x3x3 Cube, it should be publicly mentioned in the competition's information.
+   3. If the main event of the competition is not the 3x3x3 Cube, it should be publicly mentioned in the competition's information section of the 'General info' tab.
+      1. The main event should be recognizable in the competition schedule and having an appropriate proportion of rounds and attempts.
```

Comment:
Would it be a requirement that the main event is an official WCA sanctioned event?

Reply:
It isn't possible to have an unofficial event as the main event as the WRT would then not have anything to make a post about when announcing the winner. 
I realise now that this is implying that there needs to be a main event, which isn't the case (i.e. there may be no main event).

Reply:
Thanks for the clarification. I would suggest the WCAT to explicitly say that only official events are eligible to be the main event, on the WCRP. I can easily see organizers trying to have a [SEE event](https://speedcubingextraevents.org) as the main event.

Reply:
I clarified this now.